vault/request_handling.go

```diff
@@ -244,6 +250,14 @@ func (c *Core) handleRequest(req *logical.Request) (retResp *logical.Response, r
 		}
 	}
 
+	if resp != nil &&
+		req.Path == "cubbyhole/response" &&
```

Comment:
If that ever happens, we're in trouble for more than just this statement :-)

I think if we need to version this we're more likely to do `cubbyhole/response.v2` or so.